src/Codeception/Util/Drupal/FormField.php

```diff
@@ -90,15 +90,16 @@ public function __toString() {
   /**
    * Returns xpath of current identifiers element.
    *
-   * @param string $name
+   * @param string $element
    *   Name of element.
    *
    * @return string
    *   Returns path with current identifier plus requested subfield.
    */
-  public function __get($name) {
+  public function __get($element = '') {
+    $suffix = $element ? '-' . $this->normalise($element) : '';
     return $this->getXpath([
-      'identifier' => $this->getCurrentIdentifier() . '-' . $this->normalise($name),
+      'identifier' => $this->getCurrentIdentifier() . $suffix,
```

Comment:
@guncha25  Calling `$i->selectOptionFromList(FormField::field_my_list(), 'foo', '');` would result in identifier `//*[@data-drupal-selector="edit-field-list-int-0-"]` without this fix. The last parameter (target) needs to support empty values, since custom List fields usually don't have a target.